website/server/libs/pushNotifications.js

```diff
@@ -12,37 +11,27 @@ const FCM_API_KEY = nconf.get('PUSH_CONFIGS:FCM_SERVER_API_KEY');
 
 const fcmSender = FCM_API_KEY ? new gcmLib.Sender(FCM_API_KEY) : undefined;
 
-let apn;
-
+let apnProvider;
 // Load APN certificate and key from S3
 const APN_ENABLED = nconf.get('PUSH_CONFIGS:APN_ENABLED') === 'true';
 const S3_BUCKET = nconf.get('S3:bucket');
 
 if (APN_ENABLED) {
-  Promise.all([
-    S3.getObject({
-      Bucket: S3_BUCKET,
-      Key: 'apple_apn/cert.pem',
-    }).promise(),
-    S3.getObject({
-      Bucket: S3_BUCKET,
-      Key: 'apple_apn/key.pem',
-    }).promise(),
-  ])
-    .then(([certObj, keyObj]) => {
-      let cert = certObj.Body.toString();
-      let key = keyObj.Body.toString();
+  S3.getObject({
+    Bucket: S3_BUCKET,
+    Key: 'apple_apn/APNsAuthKey.p8',
+  }).promise().then((data) => {
+    const key = data.Body.toString();
```

Comment:
how long is this key going to be? We used to store it on S3 because the certificate exceeded the limit for env variables but if it's a shorter string you can add it directly to config.json.example

Reply:
I'm not actually seeing anywhere that puts a hard limit on size for these, but this issue at least implies that it should work: https://github.com/node-apn/node-apn/issues/594

Reply:
The limit was in our web server